vpr/src/base/vpr_constraints_serializer.h

```diff
@@ -298,16 +319,16 @@ class VprConstraintsSerializer final : public uxsd::VprConstraintsBase<VprConstr
     }
 
     //temp data for loads
```

Comment:
comment is out of date

Reply:
comment what it is, what it does for load and write cases.
Just have two constructors so you don't have to pass in an empty object for the load case.